### DISCUSSION TITLE
Group Assignment #4📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#21] [Assignments]
# Survey Diagram and Website Wire-frame
## Due 5/25

Submit your [Group Assignment #4](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week6/group_assignment/) by having a group member comment with the following:
- Link or photos for the survey diagram 
- Link or photos for the website wire-frame.

**Comment by kmtsujimoto** [comment]
Combined Link: https://docs.google.com/presentation/d/10AU7DSo49unuz2ZDak79Ap05Njz5EXpgKHwJ7W0TVNk/edit?usp=sharing

**Comment by czhangy** [comment]
Survey Diagram: https://www.figma.com/file/p1KITMIoTCQxBd14K80b9P/Mapping-College-Prep-Inequality-Across-LA-Survey-Wireframe?type=whiteboard&node-id=0%3A1&t=zpYq4Nbz163zuWtd-1

Wireframe:
![240798904-af68603e-90aa-444c-a705-48d021eeb763](https://github.com/albertkun/23S-ASIAAM-191A/assets/72466710/f7ab5efd-1ac2-420e-ba84-c11fa6d47cd5)

**Comment by maizaha** [comment]
[Lofi Wireframes.pdf](https://github.com/albertkun/23S-ASIAAM-191A/files/11561117/Lofi.Wireframes.pdf)
[Asia Am 191A Survey Diagram.pdf](https://github.com/albertkun/23S-ASIAAM-191A/files/11561121/Asia.Am.191A.Survey.Diagram.pdf)

**Comment by SageLuong** [comment]
Survey Diagram: 
![Survey Diagram](https://github.com/albertkun/23S-ASIAAM-191A/assets/83676556/d1263203-d319-4bc7-8e8d-f6b393a75019)

Wireframe: 
![final wireframe (final draft)](https://github.com/albertkun/23S-ASIAAM-191A/assets/83676556/6da160a2-8538-4ae2-b4bd-71635897fa95)

**Comment by chrysanthemma** [comment]
Survey Diagram
![AA191A-Survey Diagram drawio](https://github.com/albertkun/23S-ASIAAM-191A/assets/130104603/21fd86d8-9245-4959-851e-c0cbcb190f9c)

WireFrame: 
![AA191A-WireFrame drawio](https://github.com/albertkun/23S-ASIAAM-191A/assets/130104603/e950aadd-d93e-4f0f-ba48-9aabf8c0e072)

**Comment by jyjyl** [comment]
Survey Diagram:
![newStudent Housing 4 All drawio](https://github.com/albertkun/23S-ASIAAM-191A/assets/111466957/9ba2e9ef-b9ae-448f-8294-5db74c0a999f)
Wireframe:
![newWireframe Studen Housing 4 All drawio (1)](https://github.com/albertkun/23S-ASIAAM-191A/assets/111466957/ba2f99d6-ee85-4da0-a1d5-fee34eac473d)

**Comment by alanamalika** [comment]
![14A561F1-2E53-4B5C-BF8E-28FC350CEFEF](https://github.com/albertkun/23S-ASIAAM-191A/assets/97417446/4de2c3b3-797c-49c2-b94a-62810713fad1)
![BF431EFD-E810-4D3F-A98A-52FEDD77327A](https://github.com/albertkun/23S-ASIAAM-191A/assets/97417446/5af1e7eb-2c13-4686-97c5-d5134a7e717c)
https://drive.google.com/file/d/1xC00pKIpzcaV7pF-i4GFMTY4ohTi1_b3/view?usp=sharing

**Comment by emai14** [comment]
![IMG_0845](https://github.com/albertkun/23S-ASIAAM-191A/assets/130105915/e26c7433-42e3-4dd1-ad5f-29d7e0794bb1)
<img width="655" alt="Screen Shot 2023-05-25 at 3 25 19 PM" src="https://github.com/albertkun/23S-ASIAAM-191A/assets/130105915/b129e53b-1333-4628-ad74-d7aa843c1bd4">

**Comment by danigold1020** [comment]
Oil Spills:
[Wireframe.pdf](https://github.com/albertkun/23S-ASIAAM-191A/files/11594208/Wireframe.pdf)

[Survey Diagram.pdf](https://github.com/albertkun/23S-ASIAAM-191A/files/11594209/Survey.Diagram.pdf)
